test/unit/mem/view/src/ViewStaticAccMem.cpp

```diff
@@ -35,16 +35,18 @@ ALPAKA_STATIC_ACC_MEM_CONSTANT Elem g_constantMemory2DUninitialized[3][2];
 struct StaticDeviceMemoryTestKernel
 {
     ALPAKA_NO_HOST_ACC_WARNING
-    template<typename TAcc, typename TElem>
-    ALPAKA_FN_ACC void operator()(TAcc const& acc, bool* success, TElem const* const pConstantMem) const
+    template<typename TAcc, typename TMemoryHandleBool, typename TMemoryHandleElem, typename TElem>
+    ALPAKA_FN_ACC void operator()(
+        TAcc const& acc,
+        alpaka::Accessor<TMemoryHandleBool, bool, alpaka::Idx<TAcc>, 1, alpaka::WriteAccess> const success,
+        alpaka::Accessor<TMemoryHandleElem, TElem, alpaka::Idx<TAcc>, 2, alpaka::ReadAccess> const constantMem) const
```

Comment:
This raises an interesting question: Is there a way to place this memory into CUDA's/HIP's `__constant__` memory?

Reply:
Yes, it is exactly demonstrated in this file. Look at `ALPAKA_STATIC_ACC_MEM_CONSTANT Elem g_constantMemory2DInitialized[3][2]` and the call to `createStaticDevMemView`.